src/blockchain_db/blockchain_db.cpp

```diff
@@ -100,10 +100,10 @@ BlockchainDB *new_db(const std::string& db_type)
 {
   if (db_type == "lmdb")
     return new BlockchainLMDB();
-#if defined(BERKELEY_DB)
-  if (db_type == "berkeley")
-    return new BlockchainBDB();
-#endif
+//#if defined(OPTIONAL_DB)
+//  if (db_type == "optional")
+//    return new BlockchainODB();   ///Indicate right type of blockchain
+//#endif
```

Comment:
I'd rather just remove all the logic that allows for db switching instead of commenting out some pseudo code to indicate how db switching could be achieved.